### ISSUE TITLE
Why GoogLeNet v3 with auxiliary output layer got worse result than v3 without auxiliary layer

### ISSUE THREAD

**Issue by zhandongdong** [Microsoft/CNTK#1208]
I tried to use GoogLeNet v3 to train flickr style classification model which has 80K images and 20 labels. I know the images dataset is too small for v3 model. But I am confused that when auxiliary output was closed, the loss decreased and accuracy increased. Does anyone know the possible reason?

        model    = InceptionV3Norm(features, labelDim, bnTimeConst)
        z        = model.z
        aux      = model.aux

        # close the ceAux and only use ceZ got better result
        ceAux    = CrossEntropyWithSoftmax     (labels, aux)
        ceZ      = CrossEntropyWithSoftmax     (labels, z)
        errs     = ClassificationError         (labels, z)
        top5Errs = ClassificationError         (labels, z, topN=5)  # only used in Eval action
        ce       = auxWeight .* ceAux + ceZ

**Comment by cha-zhang** [comment]
Multi-task learning generally reduces the training accuracy of each task because it needs to compromise among multiple targets. Nevertheless, the test error might be better due to better generalizability.
